src/main/resources/META-INF/jqassistant-rules/spring-batch.xml

```diff
@@ -4,7 +4,7 @@
         <description>Labels all types implementing the interface "org.springframework.batch.item.ItemProcessor" with "Spring", "Batch" and "ItemProcessor".</description>
         <cypher><![CDATA[
             MATCH
-                (itemProcessor:Type)-[:EXTENDS|IMPLEMENTS*0..]->(interface:Type)
+                (artifact:Artifact)-[:CONTAINS]->(itemProcessor:Type)-[:EXTENDS|IMPLEMENTS*0..]->(interface:Type)
```

Comment:
I think we should solve it differently by removing `*0..` from the relation, thus stating that everything that implements that interface is labeled:

```
(itemProcessor:Type)-[:EXTENDS|IMPLEMENTS*]->(interface:Type)
```

Using `(artifact:Artifact)-[:CONTAINS]->` would still return the interface if we need to scan the Spring artifacts as well within a project. So it should be solved on type level, making the concept independent of any project specific build system or scan configurations.